website/routes.py

```diff
@@ -70,6 +71,19 @@ def set_status_message(user):
             extra={}
         )
 
+def anonymize_ip(request):
+    remote_addr = request.remote_addr
```

Comment:
Wasn't sure whether to try to get the IP from the flask request or try to get it in javascript.   I am seeing that remote_addr might not always be what I want?

Reply:
javascript would be nice, however I believe it can be very hard to do w/o hacks.

Reply:
the request ip will work, it should be correct on production, but i would search around the OSF there might be an example already.